build.sbt

```diff
@@ -49,22 +58,20 @@ lazy val commonSettings = Seq(
     url("https://github.com/guardian/support-frontend"),
     "scm:git:git@github.com:guardian/support-frontend.git"
   )),
-  // https://www.scala-sbt.org/1.x/docs/Cached-Resolution.html
-  updateOptions := updateOptions.value.withCachedResolution(true),
-  dependencyStats / aggregate := false,
-  dependencyTree / aggregate := false
 )
 
 lazy val commonDependencies = Seq(
   "com.typesafe" % "config" % "1.3.2",
-  "org.scalatest" %% "scalatest" % "3.2.0" % "it, test",// if the following PR is merged in v3.2.1, remove "-eU" parameter from options above - PR https://github.com/scalatest/scalatest/pull/1842
+  scalatest % "test",
   "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2"
 )
 
 lazy val root = (project in file("."))
+  .disablePlugins(ReleasePlugin, SbtPgp, Sonatype, AssemblyPlugin)
   .settings(
     name := "support-frontend-root",
-    moduleName := "support-frontend-root"
+    moduleName := "support-frontend-root",
+    publish / skip := true,
```

Comment:
moved from above, it was "loose" in the sbt file. 
Ideally for clarity, this root sbt file would just define relationships between the modules, and the modules themselves would defined any details about themselves.  But that isn't desirable to be done in this PR.